GET_STARTED.md

```diff
@@ -0,0 +1,390 @@
+ ...
+ # CCF Network initialization needed before the interaction with the service
+```
+
+Start in a CCF Network in Virtual mode, based on virtual config file: "./config/cchost_config_virtual_js.json":
+```bash
+ docker build -t ccf-app-samples:js-virtual -f docker/ccf_app_js.virtual .
+ docker run -d ccf-app-samples:js-virtual
+ ...
+ # CCF Network initialization needed before the interaction with the service
+```
+
+##### CCF Node Configuration file
+The configuration for each CCF node must be contained in a single JSON configuration file like [cchost_config_enclave_js.json - cchost_config_virtual_js.json], [ CCF node config file documentation](https://microsoft.github.io/CCF/main/operations/configuration.html)
+
+##### CCF network initialization
+After the container run, a network is started with one (node - member), you need to execute the following governance steps to initialize the network, [check Network governance section](#network-governance)
+- Activate the network members (to begin network governance)
+- Add users (using proposal)
+- Deploy the application (using proposal)
+- Open the network for users (using proposal)
+
+#### Testing: Using Linux Machine
+To Start a test CCF network on a VM, it requires [CCF to be intalled](https://microsoft.github.io/CCF/main/build_apps/install_bin.html)
+
+Start the CCF network using the cchost in release mode
+```bash
+ /opt/ccf/bin/cchost --config ./config/cchost_config_enclave_js.json
+```
+Or virtual mode
+```bash
+/opt/ccf/bin/cchost --config ./config/cchost_config_virtual_js.json
+```
+
+##### CCF Node Configuration file
+The configuration for each CCF node must be contained in a single JSON configuration file like [cchost_config_enclave_js.json - cchost_config_virtual_js.json], [ CCF node config file documentation](https://microsoft.github.io/CCF/main/operations/configuration.html)
+
+##### CCF network initialization
+Network is started with one (node - member), you need to execute the following governance steps to initialize the network,, [check Network governance section](#network-governance)
+- Activate the network members (to start a network governance)
+- Add users (using proposal)
+- Deploy the application (using proposal)
+- Open the network for users (using proposal)
+
+### Testing: Application Endpoints  
+
+Now you can send requests to your endpoints, the following sample requests to log application [(ccf-app-template)](https://github.com/microsoft/ccf-app-template/tree/main/js)
+
+In another terminal:
+
+#### Log application
+```bash
+ # send log message to be saved
+ curl -X POST https://127.0.0.1:8000/app/log?id=1 --cacert ./workspace/sandbox_common/service_cert.pem -H "Content-Type: application/json" --data '{"msg": "hello world"}'
+ # retrieve log message
+ curl https://127.0.0.1:8000/app/log?id=1 --cacert ./workspace/sandbox_common/service_cert.pem
+ # return:> hello world
+```
```

Comment:
The log application endpoints don't exist in the app-samples repo. If we push this into app-samples, it may cause confusion.
Is this just for the draft or maybe a mistake?

Reply:
Removed